project.clj

```diff
@@ -40,12 +40,15 @@
                     ;; TODO: replicate koacha's version filter logic here
                     :test-selectors {:default (complement :min-java-version)}
                     :aliases {"test" "test2junit"}}
+             :junixsocket {:jvm-opts ["-Dnrepl.test.junixsocket=true"]
```

Comment:
⚠️ I belive that the new profile, which is a very nice addition, is not exercised in CircleCI currently.

Note that we use `make test` which doesn't use the `:test-all` alias.

While we're there we might want to re-review the PR and make sure that all code paths (depending on the JDK and available classes) are exercised.

@rlbdv , what do you think?

Reply:
I think we should test it there -- I'd certainly intended to, and I have been here.  I'll see about it.